src/components/molecules/MarketStats.tsx

```diff
@@ -59,23 +82,11 @@ export default function MarketStats(): ReactElement {
 
   return (
     <div className={styles.stats} ref={ref}>
-      Total of <strong>{stats?.datasets.total}</strong> data sets & unique
-      datatokens published by <strong>{stats?.owners}</strong> accounts.
-      <br />
-      <PriceUnit
-        price={`${stats?.ocean}`}
-        small
-        className={styles.total}
-        conversion
-      />{' '}
-      and{' '}
-      <PriceUnit
-        price={`${stats?.datatoken}`}
-        symbol="datatokens"
-        small
-        className={styles.total}
-      />{' '}
-      in <strong>{stats?.datasets.pools}</strong> data set pools.
+      <Conversion price={`${totalLockedValue}`} hideApproximationSign />{' '}
+      <abbr title="Total Value Locked">TVL</abbr> (total value locked) across{' '}
+      <strong>{stats?.datasets.pools}</strong> data set pools that contain{' '}
+      <PriceUnit price={totalOceanLiquidity} small className={styles.total} />{' '}
+      and datatokens for each pool.
       <br />
       <strong>{stats?.datasets.none}</strong> data sets have no price set yet.
```

Comment:
would also kill this line completely since it doesn't really have any value, and we remove another need to fetch `market-stats` endpoint